docs/encyclopedia/nexus-services.mdx

```diff
@@ -0,0 +1,62 @@
+---
+id: nexus-services
+title: Nexus Services
+sidebar_label: Nexus Services
+description: Nexus Services are named collections of arbitrary-duration Nexus Operations that provide a microservice contract suitable for sharing across team boundaries.
+toc_max_heading_level: 4
+slug: /nexus/services
+keywords:
+  - Nexus Services
+  - contract
+  - service contract
+  - API contract
+  - microservice contract
+tags:
+  - Nexus
+  - Concepts
+
+---
+
+:::tip SUPPORT, STABILITY, and DEPENDENCY INFO
+
+Temporal Nexus is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for [Temporal Cloud](/cloud/nexus) and [self-hosted deployments](/production-deployment/self-hosted-guide/nexus).
+
+:::
+
+## A named collection of Nexus Operations
+
+A [Nexus Service](/glossary#nexus-service) is a named collection of arbitrary-duration [Nexus Operations](/nexus/operations) that provide a microservice contract suitable for sharing across team boundaries.
```

Comment:
Drop the "microservice"?

Reply:
in this context, yes will update

outside of this context IMO we still need a qualifier -- it's come up before "will users know what a service contract is, generally?" I think most devs would get it, but "microservice contract" seems to have the best SEO (if you don't know and search for it). "microservice contract" also narrows the scope vs. an unqualified "contract" or more broad "service contract".